__init__.py

```diff
@@ -127,7 +127,10 @@ def logfile_namer(default_name):
 # NOTE: This is the addon's own logger. Use it to log messages on different levels.
 # create logger
 LookingGlassAddonLogger = logging.getLogger('Alice/LG')
-LookingGlassAddonLogger.setLevel(logging.DEBUG)
+if LookingGlassAddon.debugging_print_internal_logger_all:
+	LookingGlassAddonLogger.setLevel(logging.DEBUG)
+else:
+	LookingGlassAddonLogger.setLevel(logging.ERROR)

```

Comment:
I would keep the original here, because the `LookingGlassAddon.debugging_print_internal_logger_all` is only for console output and I'd rather not mix it with the logger for the log files.

```suggestion
LookingGlassAddonLogger.setLevel(logging.DEBUG)
```

Reply:
You'll see I reverted this change in the following commit, however the result is there actually are printouts.

The current behavior on this branch with my latest commit in:
```
[Alice/LG] [INFO] 12/23/2021 13:36:13 - ----------------------------------------------
[Alice/LG] [INFO] 12/23/2021 13:36:13 - Initializing 'Alice/LG v2.0.5' ...
[Alice/LG] [INFO] 12/23/2021 13:36:13 -  [#] Add-on path: /Users/patrickcrawford/Library/Application Support/Blender/3.0/scripts/addons/AliceLG
[Alice/LG] [INFO] 12/23/2021 13:36:17 -  [#] Registered add-on operators in Blender.
[Alice/LG] [INFO] 12/23/2021 13:36:17 -  [#] Done.
[Alice/LG] [INFO] 12/23/2021 13:36:17 - Connecting to HoloPlay Service ...
[pyLightIO] [ERROR] 12/23/2021 13:36:17 - Could not connect. Is HoloPlay Service running?
[pyLightIO] [WARNING] 12/23/2021 13:36:17 - Added service 'HoloPlay Service' to the service manager, but service is not ready.
[Alice/LG] [INFO] 12/23/2021 13:36:17 -  [#] Connection failed.
```

But the behavior when `LookingGlassAddonLogger.setLevel(logging.DEBUG)` is replaced with the logic above, is that it then only prints out:

```
[pyLightIO] [ERROR] 12/23/2021 13:34:19 - Could not connect. Is HoloPlay Service running?
[pyLightIO] [WARNING] 12/23/2021 13:34:19 - Added service 'HoloPlay Service' to the service manager, but service is not ready.
```

Not sure if this is being complicated by the registration order, or if another update needs to be made to the logger such that the turn off logging to stdout sooner. Again not a huge deal, it's just 9 extra lines of printouts whenever blender starts, my ocd just wants to minimize that where possible!

Reply:
Yeah, I understand that. The problem is that you also mute the output to the log files with that approach. I would keep it like that at the moment and when I rework the initialization of the add-on in Alice/LG 2.1, the console output will be completely muted. Your pull request here is a good preparation for that, so thanks a lot!